grafana/provisioning/dashboards/dashboard.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: 1
```

Comment:
Can you add a short comment on top of this file to help readers understand its purpose?

Reply:
Hm, the dashboard that we wanna keep is `tools/dashboard.json` that is removed by https://github.com/topos-protocol/topos/pull/397/commits/121e330a944f9680bcb0dee25d69c67447f5d11e

Then I believe you can `mv tools/dashboard.json grafana/benchmarks-dashboard.json`

As for the current ones in the `provisioning` folder, I think we can remove this folder completely as it looks like default dashboards that already come with our grafana deployments